### ISSUE TITLE
Key binding for Shift-K

### ISSUE THREAD

**Issue by kenk-ccs** [jtkDvlp/web-mode-edit-element#7]
The keybinding `("C-K" web-mode-edit-element-elements-kill-siblings)` doesn't work under emacs 26.1 (perhaps others but I haven't tested). The capital K isn't interpreted as a shift-k chord; emacs sees it as a plain k and thus maps ctrl-k to `web-mode-edit-element-elements-kill-siblings`. The fix is to specify the chord explicitly: `("C-S-k" web-mode-edit-element-elements-kill-siblings)`

**Comment by jtkDvlp** [comment]
Thank you for your issue. Now it works for me.

Feel free to reopen the issue if it´s not working for you.

**Comment by kenk-ccs** [comment]
The change should be applied to the M-K binding as well: M-S-K

**Comment by jtkDvlp** [comment]
Yeah, I also expected that but control and meta bindings behave different. Do you have tried the M-K and M-k, for me it works?

**Comment by kenk-ccs** [comment]
It wasn't working until I mapped it via M-S-k locally

**Comment by jtkDvlp** [comment]
Mhh ok, I will research about this.
